packages/fx-core/tests/plugins/resource/aad/unit/expectedBicepFiles/bot/variables.bicep

```diff
@@ -0,0 +1,2 @@
+
+var applicationIdUri = `api://test_bot_domain_url/botid-${BotClientId}`
```

Comment:
maybe no need to create a separate folder? since it only contains one file. We may just treat bot as file name prefix

Reply:
Fxied.